website/source/docs/agent/checks.html.md

```diff
@@ -10,7 +10,8 @@ description: |-
 
 One of the primary roles of the agent is management of system-level and application-level health
 checks. A health check is considered to be application-level if it is associated with a
-service. If not associated with a service, the check monitors the health of the entire node.
+service. Review the [Getting Started guide](https://learn.hashicorp.com/consul/getting-started/services?utm_source=consul.io&utm_medium=docs) for a complete example of a application-level health check. 
```

Comment:
```suggestion
service. Review the [Getting Started guide](https://learn.hashicorp.com/consul/getting-started/services?utm_source=consul.io&utm_medium=docs) on service registration for a complete example of how to add and application-level health check to a service registration file.
```

Reply:
This was a bit of a weird experience because the subtitle in the learn guide, where we add thee check, is called "Updating Services" and I was skimming for something about health checking. We may want to change that subtitle in the guide.